src/components/Notification.tsx

```diff
@@ -0,0 +1,60 @@
+import { useCallback, useEffect } from 'react';
+import { Alert, Link, Snackbar, styled } from '@mui/material';
+
+import { NotificationActionTypes } from '~/providers/NotificationProvider/actions';
+import { useCustomTheme, useNotificationContext } from '~/hooks';
+
+export default function Notification() {
+  const { state, dispatchNotification } = useNotificationContext();
+  const { type, message, visible, timeout, link } = state;
+
+  const clearNotification = useCallback(() => {
+    dispatchNotification({ type: NotificationActionTypes.CLEAR_NOTIFICATION });
+  }, [dispatchNotification]);
+
+  useEffect(() => {
+    let timeoutId: NodeJS.Timeout;
+    if (visible && timeout > 0) {
+      timeoutId = setTimeout(() => {
+        clearNotification();
+      }, timeout);
+    }
+
+    return () => clearTimeout(timeoutId);
+  }, [timeout, visible, clearNotification, state]);
+
+  const severity = type === 'loading' ? 'info' : type;
+
+  return (
+    <Snackbar
+      open={visible}
+      autoHideDuration={timeout}
+      message={message}
+      anchorOrigin={{ horizontal: 'right', vertical: 'top' }}
+      sx={{ position: 'absolute' }}
```

Comment:
we recommend avoiding adding inline styles

Reply:
https://github.com/0xShishigami/feo-web3-react-boilerplate/pull/2/commits/43b247bae6f8134af1759dfc628d6ace34fd53b3